src/test/java/com/puppycrawl/tools/checkstyle/bdd/InlineConfigParser.java

```diff
@@ -649,11 +655,8 @@ else if (violationBelowWithExplanationMatcher.matches()) {
             inputConfigBuilder.addViolation(violationLineNum, violationMessage);
         }
         else if (violationWithExplanationMatcher.matches()) {
-            final String violationMessage = violationWithExplanationMatcher.group(1);
             final int violationLineNum = lineNo + 1;
-            checkWhetherViolationSpecified(specifyViolationMessage, violationMessage,
-                    violationLineNum);
-            inputConfigBuilder.addViolation(violationLineNum, violationMessage);
+            inputConfigBuilder.addViolation(violationLineNum, null);
```

Comment:
Why was this changed?

Reply:
To fix a problem that we started from. Without it migration of example can not happen

Reply:
This doesn't make things clearer. We said we missed violation from bad violation text in input and changed regexp to match. Why does this need to change as well?

Reply:
comparison of 
https://github.com/checkstyle/checkstyle/blob/e071377c4d6e049491aef610b04bc106e8a8b6a3/src/test/java/com/puppycrawl/tools/checkstyle/bdd/InlineConfigParser.java#L80-L81
vs
https://github.com/checkstyle/checkstyle/blob/e071377c4d6e049491aef610b04bc106e8a8b6a3/src/test/java/com/puppycrawl/tools/checkstyle/bdd/InlineConfigParser.java#L100-L101
```
.compile(".*//\\s*violation,?\\s*(?:['\"](.*)['\"])?$")
.compile(".*//\\s*violation,\\s.+\\s(?:['\"](.*)['\"])?$");

with extra spaces (to spot a difference):
.compile(".*//\\s*violation,?\\s*   (?:['\"](.*)['\"])?$")
.compile(".*//\\s*violation,\\s.+\\s(?:['\"](.*)['\"])?$");
```

so VIOLATION_WITH_EXPLANATION_PATTERN kind of copy paste, and it demand message `'some text' or "some text"` to be present, but role of it to allow any text without `"` or `'`

we changing VIOLATION_WITH_EXPLANATION_PATTERN to do no message match, and handling of this pattern should have null in violation message parameter - no matching by message will happen.

Reply:
1) If `VIOLATION_WITH_EXPLANATION_PATTERN` no longer has an explanation matching, then we need to remove it from the field name.
2) I thought we wanted violation message matching as part of BDD? I remember seeing an issue to convert other inputs to this format. Others have violation message matching and it seems like this is the only one who won't, so again, why are we doing this?